README.md

```diff
@@ -14,10 +18,57 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+If you want to install Node.js natively visit [Node.js](http://nodejs.org), please make sure it is version 4.4.4
+
+To check the locally installed version run: 
+
+```bash
+node --version
 ```
-git clone https://github.com/department-of-veterans-affairs/healthcare-application.git
-cd healthcare-application
+
+Once you have version 4.4.4 of Node.js installed to required version of npm run:
+
+```bash
+npm i -g npm@3.8.9
+```
+
+You may want to use nvm (Node Version Manager), this will allow for managing and installing multiple versions of Node.js on your machine.
```

Comment:
I feel like these are 2 different workflows: either you install Node natively or you use nvm to do it. And then these workflows come together to install npm and the project's dependencies. Let's make that a bit more clear in the documentation. Here's the flow of work as I see it:
1. First, check if you have Node installed
2. If not, you have 2 choices for how to install: natively or using nvm
3. Instructions on how to install Node natively
4. Instructions on how to install Node using nvm
5. Once you have Node installed, require the correct version of npm
6. Install project dependencies

We can use a list and/or sub-headers to clearly identify these steps.